src/main/java/org/springframework/test/context/junit5/SpringParameterExtension.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.test.context.junit5;
+
+import org.junit.gen5.api.extension.ExtensionContext;
+import org.junit.gen5.api.extension.MethodInvocationContext;
+import org.junit.gen5.api.extension.MethodParameterResolver;
+import org.junit.gen5.api.extension.ParameterResolutionException;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.TestContext;
+import org.springframework.test.context.TestContextManager;
+import org.springframework.util.ReflectionUtils;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+
+/**
+ * @author Tadaya Tsuyukubo
+ */
+public class SpringParameterExtension extends SpringExtension implements MethodParameterResolver {
+
+	@Override
+	public boolean supports(Parameter parameter, MethodInvocationContext methodInvocationContext, ExtensionContext extensionContext) throws ParameterResolutionException {
+		return true;  // TODO: should check
```

Comment:
This is actually forbidden in JUnit 5 extensions (even if not documented). So we definitely cannot simply return `true`. :wink: